saleor/graphql/product/types/products.py

```diff
@@ -332,13 +370,16 @@ def resolve_quantity_ordered(root: models.ProductVariant, *_args):
         # `resolve_report_product_sales` resolver.
         return getattr(root, "quantity_ordered", None)
 
-    @staticmethod
     @permission_required(ProductPermissions.MANAGE_PRODUCTS)
-    def resolve_quantity_allocated(root: models.ProductVariant, *_args):
-        return root.quantity_allocated
+    def resolve_quantity_allocated(root: models.ProductVariant, info):
+        country = info.context.country
+        stock = stock_models.Stock.objects.get_variant_stock_for_country(country, root)
+        return stock.quantity_allocated
 
     @staticmethod
-    @permission_required(ProductPermissions.MANAGE_PRODUCTS)
+    @permission_required(
+        [OrderPermissions.MANAGE_ORDERS, ProductPermissions.MANAGE_PRODUCTS]
```

Comment:
I think we've changed that on master to only require `MANAGE_PRODUCTS` permission.

Reply:
@maarcingebala Fixed